L1Trigger/L1THGCal/python/customClustering.py

```diff
@@ -80,15 +80,29 @@ def custom_3dclustering_histoMax(process,
         nBins_R = 36,
         nBins_Phi = 216,
         binSumsHisto = binSums,                        
+        seed_threshold = 0,
         ):
     parameters_c3d = process.hgcalBackEndLayer2Producer.ProcessorParameters.C3d_parameters
     parameters_c3d.dR_multicluster = cms.double(distance)
     parameters_c3d.nBins_R_histo_multicluster = cms.uint32(nBins_R)
     parameters_c3d.nBins_Phi_histo_multicluster = cms.uint32(nBins_Phi)
     parameters_c3d.binSumsHisto = binSumsHisto
+    parameters_c3d.threshold_histo_multicluster = seed_threshold
     parameters_c3d.type_multicluster = cms.string('HistoMaxC3d')
     return process
 
+def custom_3dclustering_histoMax_variableDr(process,
```

Comment:
Could you just modify the dR parameter in this function and not the seeding? 
The goal in the near future would be to disentangle the seeding customisation and the clustering customisation.

Reply:
that would be fine for me, although it would break the backwards compatibility